prow/config/config.go

```diff
@@ -1193,17 +1195,23 @@ func ReadJobConfig(jobConfig string) (JobConfig, error) {
 		}
 		uniqueBasenames.Insert(base)
 
+		begin := time.Now()
```

Comment:
could we say something like `allStart` + `fileStart` instead of `start/begin` for reability?

Reply:
Done!